content/applications/productivity/to_do.rst

```diff
@@ -0,0 +1,134 @@
+  new tag simply by writing it;
+- :guilabel:`Assignees`: select one or more users from the dropdown list to assign your to-do;
+- **Customize your tasks content**: when typing `/` in the text area, a dropdown list of features
+  related to the structure, format, media, navigation, widgets, banner, or basic blocks appear;
+- :ref:`Personal stages <to-do/personal-stages>`: move your task to another stage by clicking on
+  your desired stage;
+- :guilabel:`Mark as done`: click the check mark button when the task has been completed.
+
+Click on the gear button (:guilabel:`⚙`) to access these actions :guilabel:`Archive`,
+:guilabel:`Duplicate`, :guilabel:`Delete`, :guilabel:`Dashboard`, or :ref:`Convert to task
+<to-do/convert-task>`.
+
+.. note::
+   To-dos, like private tasks, are only accessible to the users specified as assignees. You can use
+   the :guilabel:`Assignees` field to share a to-do with other users.
+
+.. image:: to_do/task-edition.png
+   :alt: edit your task
+
+.. _to-do/personal-stages:
+
+Using stages
+------------
+
+Your assigned **to-do tasks** are displayed on your dashboard in different default stages. You can
+drag and drop a to-do to move it from one stage to another.
+
+Click the :guilabel:`+ Personal Stages` button to create a new stage.
+
+.. note::
+   Click the gear button (:guilabel:`⚙`) on a specific stage to :guilabel:`Fold`, :guilabel:`Edit`,
+   or :guilabel:`Delete` the stage.
+
+    .. image:: to_do/fold-edit-delete.png
+       :alt: Settings button
+
+Scheduling activities
+=====================
+
+To schedule an activity from a to-do, click on :guilabel:`Activities`; or from the dashboard, click
+the clock icon on a to-do card, then :guilabel:`+ Schedule an activity`:
+
+.. image:: to_do/schedule-activity.png
+   :alt: schedule an activity
+
+Then, fill in the required fields:
+
+- :guilabel:`Activity Type`: select the type of activity from the list;
+- :guilabel:`Summary`: add a brief description of the activity;
+- :guilabel:`Due date`: select the date the activity is due;
+- :guilabel:`Assigned to`: select an assignee for the activity;
+- :guilabel:`Log a note`: you can add further description to the activity.
+
+Click :guilabel:`Schedule` to complete the action.
+
+.. note::
+   The :guilabel:`To Do` :guilabel:`Activity Type` is different than a to-do task. Therefore, adding
+   one doesn't create a to-do.
+
+.. _to-do/convert-task:
+
+Converting to-dos into project tasks
+====================================
+
+You can convert a to-do into a task and assign it to a project by going to the to-do you want to
+convert, clicking the gear button (:guilabel:`⚙`) and :guilabel:`Convert to Task`. Select the
+:guilabel:`Project`, add or remove :guilabel:`Assignees`, select :guilabel:`Tags` from the project,
+and click :guilabel:`Convert to Task`. The to-do is now a project task and appears in the Project
+app under :guilabel:`My Tasks`.
```

Comment:
Once the to-do has been converted into a task, it appears in all of the menus of project, not only under 'my tasks'.